tests/unit/modules/network/sonic/test_sonic_logging.py

```diff
@@ -0,0 +1,87 @@
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+from ansible_collections.dellemc.enterprise_sonic.tests.unit.compat.mock import (
+    patch,
+)
+from ansible_collections.dellemc.enterprise_sonic.plugins.modules import (
+    sonic_logging,
+)
+from ansible_collections.dellemc.enterprise_sonic.tests.unit.modules.utils import (
+    set_module_args,
+)
+from .sonic_module import TestSonicModule
+
+
+class TestSonicInterfacesModule(TestSonicModule):
+    module = sonic_logging
+
+    @classmethod
+    def setUpClass(cls):
+        cls.mock_facts_edit_config = patch(
+            "ansible_collections.dellemc.enterprise_sonic.plugins.module_utils.network.sonic.facts.logging.logging.edit_config"
+        )
+        cls.mock_config_edit_config = patch(
+            "ansible_collections.dellemc.enterprise_sonic.plugins.module_utils.network.sonic.config.logging.logging.edit_config"
+        )
+        cls.mock_utils_send_requests = patch(
+            "ansible_collections.dellemc.enterprise_sonic.plugins.module_utils.network.sonic.config.logging.logging.send_requests"
+        )
+        cls.mock_get_interface_naming_mode = patch(
+            "ansible_collections.dellemc.enterprise_sonic.plugins.module_utils.network.sonic.utils.utils.get_device_interface_naming_mode"
+        )
+        cls.fixture_data = cls.load_fixtures('sonic_logging.yaml')
+
+    def setUp(self):
+        super(TestSonicInterfacesModule, self).setUp()
+        self.facts_edit_config = self.mock_facts_edit_config.start()
+        self.config_edit_config = self.mock_config_edit_config.start()
+        self.facts_edit_config.side_effect = self.facts_side_effect
+        self.config_edit_config.side_effect = self.config_side_effect
+        self.get_interface_naming_mode = self.mock_get_interface_naming_mode.start()
+        self.get_interface_naming_mode.return_value = 'standard'
+        self.utils_send_requests = self.mock_utils_send_requests.start()
+        self.utils_send_requests.side_effect = self.send_requests_side_effect
```

Comment:
Related to the question asked on the sonic_module file: What is the purpose of this additional setup step?

Reply:
This code has been revised.

I am marking this issue "Resolved".